### PR TITLE
PIC - 364 Order summary - Current custodial order requirements & logic

### PR DESCRIPTION
**Pull request by sarahyjja**
2 tests are not passing and I don't know why... I would be happy to have help on these! :)

View the probation record section of the case summary for a current offender
Screenshot 2020-09-23 at 14 32 31

View the probation record to see the post-sentence supervision requirements attached to a current custodial order.
Screenshot 2020-09-23 at 14 32 07

### CODE REVIEW

File: server/views/case-summary-record-order.njk [hunk 1]
```diff
@@ -182,6 +182,22 @@
                 </div>
             </div>
 
+            {% if currentOrder.licenceConditions | length %}
+                {% if custodialOrder.custodialTypeCode === "B" %}
+                    <h2 class="govuk-heading-m govuk-!-margin-top-8">Licence conditions</h2>
```
Comment (theDustRoom) [inline server/views/case-summary-record-order.njk:187]: You could inline this **if** to reduce a little duplication

`<h2 class="govuk-heading-m govuk-!-margin-top-8">{{"Licence conditions" if custodialOrder.custodialTypeCode === "B" else "Post-sentence supervision requirements"}}</h2>`

File: mappings/community/DX12340A-requirements-1309234876.json [hunk 2]
```diff
@@ -10,6 +10,37 @@
     },
     "status": 200,
     "jsonBody": {
+      "licenceConditions": [
```
Comment (theDustRoom) [inline mappings/community/DX12340A-requirements-1309234876.json:13]: You appear to have two "licenseConditions" in your JSON

File: server/views/case-summary-record-order.njk (hunk 1 above)
Comment (sarahyjja) [reply server/views/case-summary-record-order.njk:187]: 🙆‍♀️

File: mappings/community/DX12340A-requirements-1309234876.json (hunk 2 above)
Comment (sarahyjja) [reply mappings/community/DX12340A-requirements-1309234876.json:13]: weird, my bad!

**Review by theDustRoom — APPROVED** [review]
LGTM

**Pull request merged**